test/scylla-gdb/test_misc.py

```diff
@@ -89,6 +90,11 @@ def test_task_queues(gdb):
 def test_task_histogram(gdb):
     scylla(gdb, 'task_histogram')
 
+def test_task_histogram_coro(gdb):
+    h = scylla(gdb, 'task_histogram -a')
+    if re.search(r'\) \[clone \.\w+\]', h) is None:
```

Comment:
`if not re.search(r'\) \[clone \.\w+\]', h)` would work just as well
and is more flexible with regards to what is returned (or not) from re.search.

Reply:
This flexibility can be undesired. I the canary is `None`, i'd say the current code is better.
I had filtering code break because in addition to the `None` canary, `not filter` also disabled filtering when I filtered for `0`.